docs/conf.py

```diff
@@ -149,5 +149,8 @@ def setup(app):
 # include that variant in the sitemap.
 sitemap_url_scheme = "latest/{link}"
 
+# ignore internal urls when running linkcheck
+linkcheck_ignore = [r'^#']
```

Comment:
why ignore these?  Does it not handle plain fragments properly?

Reply:
The process would just freeze for me when I didn't ignore them. Not sure why. Also the ticket was for external links only.